tests/test_parser.py

```diff
@@ -99,6 +99,11 @@ def test_structs(self):
     def test_float(self):
         self.assertEqual(parse_one(".2"), parse_one("0.2"))
 
+    def test_bpchar(self):
+        cast = parse_one("cast(foo as bpchar)")
```

Comment:
can you delete this test and just put it into identity.sql as a fixture?

Reply:
Sure thing; done